src/MakePotCommand.php

```diff
@@ -165,6 +170,16 @@ public function __invoke( $args, $assoc_args ) {
 		WP_CLI::success( 'POT file successfully generated!' );
 	}
 
+	/**
+	 * Removes leading and trailing slashes of a string.
+	 *
+	 * @param string $string What to add the remove slashes from.
+	 * @return string String without leading and trailing slashes.
+	 */
+	protected function unslashit( $string ) {
+		return ltrim( rtrim( $string, '/\\' ), '/\\' );
```

Comment:
This would actually be a good use of `DIRECTORY_SEPARATOR`. However, as we are not quite sure what we get and what we already did with it, I'd say we leave it as is now to stay on the safe side (especially with Cygwin on Windows).